jetson/face_detector_threaded.py

```diff
@@ -236,16 +238,17 @@ def printProbs(self):
             print('\n')
 
 
-def encryptFace(coordinates, img): 
-    #Accepts list of face coordinates and img,
-    #Calls AES to encrypt region
-    #For now, returns img with encrypted face(s)
+def encryptFace(coordinates, img):
+    # Accepts list of face coordinates and img,
+    # Calls AES to encrypt region
+    # For now, returns img with encrypted face(s)
```

Comment:
please convert these comments into a docstring for this method. Also include documentation on the parameters

Reply:
@jbagnara Could you do this? I believe you wrote this code.